config/variables.tf

```diff
@@ -108,6 +108,32 @@ variable "gcs_to_bq_runner_role_id" {
   type        = string
 }
 
+# Dedupe Cloud Run Service Vars
+variable "dedupe_service_name" {
+  description = "Name of the Cloud Run service for removing "
```

Comment:
Finish comment here and update comments below to dedupe service instead of GCS-to-BQ service.

Reply:
Ah thanks for catching that.